pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -87,6 +88,15 @@ const (
 	// StateAvailable means the host can be consumed
 	StateAvailable ProvisioningState = "available"
 
+	// StateManualCleaning means we are cleaning the old configuration and set new configuration
+	StateManualCleaning ProvisioningState = "manual cleaning"
```

Comment:
Since we don't need a 1-1 mapping to ironic states, maybe rename this? The term "cleaning" is vague in this context, but it's too late to change ironic itself. Any ideas from other reviewers?